.github/actions/vmtest/action.yml

```diff
@@ -43,7 +43,10 @@ runs:
       run: |
         echo "::group::Prepare buidling selftest"
         cd .kernel
-        cp ${{ github.workspace }}/travis-ci/vmtest/configs/config-latest.${{ inputs.arch }} .config
+        cat tools/testing/selftests/bpf/config \
+            tools/testing/selftests/bpf/config.${{ inputs.arch }} \
+            ${{ github.workspace }}/travis-ci/vmtest/configs/config-latest \
+            ${{ github.workspace }}/travis-ci/vmtest/configs/config-latest.${{ inputs.arch }} 2> /dev/null > .config && :
```

Comment:
do we need :  here to ensure cat always return true?

Reply:
Yep, that is my understanding.